microsoft-edge/devtools-guide-chromium/whats-new/2023/12/devtools-120.md

```diff
@@ -0,0 +1,103 @@
+---
+title: What's new in DevTools (Microsoft Edge 120)
+description: "Edge DevTools has a new user interface. Explain with Copilot in DevTools is generally available. New Recorder tool to record and replay common user flows and measure performance. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 12/07/2023
+---
+# What's New in DevTools (Microsoft Edge 120)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## DevTools has a new user interface
+
+<!-- Subtitle: Introducing the all-new Edge DevTools user interface.  Check out our blog post to learn about the new Activity Bar, Quick View, and how to customize your developer tools in the redesigned DevTools UI. -->
+<!-- todo: the above is from Welcome 119 -->
+
+DevTools has a new look, starting with Microsoft Edge 120!  The major design updates are the new **Activity Bar** with icons; **Quick View** in place of the Drawer at bottom of DevTools; and the ability to easily open, close, and move tools around:
+
+![Updated DevTools UI](./devtools-120-images/new-devtools-ui.png)
+
+See blog post: [Inside the all-new Edge DevTools user interface](https://blogs.windows.com/msedgedev/2023/12/20/inside-the-all-new-edge-devtools-ui/).
+
+If you prefer the legacy UI, you can switch back by going to DevTools > **Settings** > **Preferences** and selecting the **Switch back to the legacy DevTools UI** checkbox.  After reloading DevTools, the legacy UI appears.
+
+
+<!-- ====================================================================== -->
+## The "Explain with Copilot in DevTools" feature is generally available
+
+<!-- Subtitle: Debug source code by having Copilot explain lines of code.  You can ask Copilot to read lines of code for you and describe what the code is doing for an easier time debugging. -->
+<!-- todo: the above is from Welcome 119 -->
```

Comment:
@leahmsft @zoherghadyali - confirm Subtitle for What's New 120 entry "The "Explain with Copilot in DevTools" feature is generally available".